### PR TITLE
Private/devidask/ft 62

### PR DESCRIPTION
**Pull request by devidask27**

### CODE REVIEW

File: cmd/prepNode.go [hunk 1]
```diff
@@ -70,10 +72,37 @@ func checkAndValidateRemote() bool {
...
+				fmt.Printf("Enter Username: ")
+				reader := bufio.NewReader(os.Stdin)
+				user, _ = reader.ReadString('\n')
+				user = strings.TrimSpace(user)
+			}
+			if sshKey == "" && password == "" {
+				/*fmt.Printf("Enter Password: ")
 				passwordBytes, _ := terminal.ReadPassword(0)
-				password = string(passwordBytes)
+				password = string(passwordBytes)*/
```
Comment (anupbarve) [inline cmd/prepNode.go:84]: Remove.

File: cmd/prepNode.go [hunk 2]
```diff
@@ -70,10 +72,37 @@ func checkAndValidateRemote() bool {
...
+				var choice int
+				fmt.Println("You can choose either password or sshKey")
+				//fmt.Println("Please enter your choice (password/ssh)")
+				fmt.Println("Enter 1 for password and 2 for sshKey")
+				fmt.Print("Enter Option : ")
+				fmt.Scanf("%d", &choice)
+				//choice = strings.ToLower(choice)
```
Comment (anupbarve) [inline cmd/prepNode.go:91]: Remove.

**Review by anupbarve — APPROVED** [review]
lgtm

**Review by anupbarve — APPROVED** [review]

**Comment by devidask27** [comment]
http://teamcity.platform9.horse:8111/buildConfiguration/Pf9project_Platform9Components_Pf9ctl/1448399?focusLine=0

**Pull request merged**